### PR TITLE
feat: using properties instead of the json

### PR DESCRIPTION
**Pull request by g-cheishvili**
## Related Issue(s)

https://github.com/SAP/fundamental-ngx/issues/10137#issuecomment-1780206126

## Description

- using properties format instead of the json
- updated nx because needed updated builders(at the time of update)
- modified application in a way that now it loads translations through http, instead of direct import
- added jsons into the ignored files list, because source of truth now is the `.properties` files. every time you start or build the application, those will be created(if needed, it's cached), but they are not created with webpack virtual modules, I wanted like that, but Angular's cache is breaking after every restart, so custom webpack config did not work as expected

### CODE REVIEW

File: libs/docs/shared/src/lib/core-helpers/toolbar/toolbar.component.ts [hunk 1]
```diff
@@ -186,6 +180,7 @@ export class ToolbarDocsComponent implements OnInit, OnDestroy {
         private _docsService: DocsService,
         private _http: HttpClient
     ) {
+        langSubject$.subscribe((lang) => console.log({ lang }));
```
Comment (N1XUS) [inline libs/docs/shared/src/lib/core-helpers/toolbar/toolbar.component.ts:183]: Is it needed?

Comment (g-cheishvili) [reply libs/docs/shared/src/lib/core-helpers/toolbar/toolbar.component.ts:183]: No, I forgot

**Review by mikerodonnell89 — COMMENTED** [review]
Unit tests are still looking for the old json files

**Review by mikerodonnell89 — COMMENTED** [review]
I'm nitpicking here, but this doc page needs a little updating for the new *.properties files: https://deploy-preview-10846--fundamental-ngx.netlify.app/#/i18n/writing-translations

**Comment by g-cheishvili** [comment]
Closing this, because I accidentally froze the CI, need fresh one

**Pull request closed without merging**